docs/source/tutorial_optimization.rst

```diff
@@ -0,0 +1,900 @@
+    report(fom_result, reference_minimization_data, reference_mu)
+    
+    print('\nROM with finite differences')
+    report(rom_result, RB_minimization_data, reference_mu)
+    
+    print('\nFOM with gradient')
+    report(opt_fom_result, opt_fom_minimization_data, reference_mu)
+    
+    print('\nROM with gradient')
+    report(opt_rom_result, opt_rom_minimization_data, reference_mu)
+    
+    print('\nAlways enrich along the path')
+    report(opt_along_path_result, opt_along_path_minimization_data, reference_mu)
+    
+    print('\nAdaptively enrich along the path')
+    report(opt_along_path_adaptively_result, opt_along_path_adaptively_minimization_data, reference_mu)
+
+.. jupyter-execute::
+    :hide-code:
+    :hide-output:
+    
+    assert fom_result.nit == 10
+    assert opt_along_path_result.nit == 8
+    assert opt_along_path_minimization_data['num_evals'] == 10
+    assert opt_along_path_minimization_data['enrichments'] == 9
+    assert opt_along_path_adaptively_minimization_data['enrichments'] == 4
+
+
+    
+
+Some general words about MOR methods for optimization
+-----------------------------------------------------
+
+This notebook has shown several aspects on how to use RB methods for
+reducing a FOM for an optimization problem. One main result from this
+was that standard RB methods can help to reduce the computational time.
+Thus, standard RB methods are especially of interest if an optimization
+problem might need to be solved multiple times.
+
+However, for only a single optimization routine, their expensive offline
+time might make them unfavorable because they lack overall efficiency.
+The example that has been discussed in this notebook is a very simple
+and low dimensional problem with a linear output functional. Especially
+going to high dimensional parameter spaces and non linear output
+functionals would aggravate this effect even more.
+
+To resolve this issue we have seen a way to overcome the traditional
+offline/online splitting and saw that it is a good idea to enrich the
+model along the path of the optimization or (even better) only enrich
+the model if the standard error estimator goes above a certain
+tolerance.
+
+Furthermore, higher order optmization methods with accessible gradient
+or hessian make FOM methods take even less steps. Also in this case,
+adaptive RB methods still reduce the computational demand of the
+optimization method.
+
+For more advanced methods and problems on this topic, we refer to
+Trust-Region methods, quadratic objective functionals, inverse problems
+or higher order optimization methods. A Trust-Region method for
+quadratic objective functionals with a non-conforming RB approach has
+been considered in `this (arXiv)
+paper <https://arxiv.org/abs/2006.09297>`__, where pyMOR has been used
+for the MOR part. You can see the whole code and all numerical results
+in `this
+project <https://github.com/TiKeil/NCD-corrected-TR-RB-approach-for-pde-opt>`__.
```

Comment:
Only citing your own preprint regarding RB methods for optimization sounds like a bit too much of self-advertisement. Please also reference some published standard literature as well.